convert2rhel/pkghandler.py

```diff
@@ -318,8 +318,7 @@ def format_pkg_info(pkgs, disable_repos=None):
     max_nvra_length = max(len(nvra) for nvra in package_info)
 
     header = (
-        "%-*s  %-*s  %s"
```

Comment:
I've got this from Grok AI since it's been long since I wrote that code:
> ```
> %-*s in print("%-*s" % (width, "text")) is a format specifier where:
>    * means the width of the field is given by the first argument in the tuple (width in this case).
>    - means left-align the text within that width.
>    s means it's formatting a string.```

The `format()` replacement would look like `print("{:<{}}".format("text", width))`.
> ```
> {:<{}} means:
>   < aligns the text to the left.
>   {} specifies the field width.```